src/uu/ls/src/ls.rs

```diff
@@ -1306,7 +1304,7 @@ pub fn uu_app() -> Command {
                 .num_args(0..=1)
                 .default_missing_value("always")
                 .default_value("never")
-                .value_name("WHEN"),
+                .value_name("WHEN").overrides_with(options::DIRED),
```

Comment:
For some reason `fmt` doesn't work in this function:
```suggestion
                .value_name("WHEN")
                .overrides_with(options::DIRED),
```